src/font-dialog.c

```diff
@@ -278,7 +283,7 @@ create_font_dialog(FontDialog *fd)
     *ls_size = gtk_list_store_new(2, G_TYPE_STRING, G_TYPE_INT);
   GtkCellLayout *cl;
   GtkCellRenderer *cr;
-  fd->dlg = GTK_DIALOG(gtk_dialog_new_with_buttons(g_dgettext("gtk20", "Pick a Font"), NULL, GTK_DIALOG_NO_SEPARATOR,
+  fd->dlg = GTK_DIALOG(gtk_dialog_new_with_buttons(g_dgettext("gtk20", "Preferances"), NULL, GTK_DIALOG_NO_SEPARATOR,
```

Comment:
Imo the dialog is a missnomer since it contains more than just the font, with the addition of the option this pr add more so than before, but even before this dialog contains the scroll behavior too. so i renamed it. ill fix the typo

Reply:
So i fixed the typo, are there any other objections to this pr still?